docker-compose-test.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.4"
+services:
+  report:
+    build:
+      context: .
+      dockerfile: Dockerfile-test
```

Comment:
Should this line be removed or is there a file that has not been commited?

Either way I think we want to test the production image really.

Reply:
Yup, this should be removed. Good spot!